components/01-atoms/videos/_video.scss

```diff
@@ -1,20 +1,22 @@
 /* Responsive Video using CSS only */
 .video {
-  height: 0;
-  overflow: hidden;
-  padding-top: 35px;
-  padding-bottom: 56.25%; /* 56.25% = 16x9 */
-  position: relative;
-
   iframe {
     height: 100%;
     left: 0;
     position: absolute;
     top: 0;
     width: 100%;
   }
+
+  video {
+    width: 100%;
+    height: auto;
+  }
 }
 
-.video--full {
-  padding-bottom: 75%; /* 75% = 4x3 */
+.video__wrap {
+  height: 0;
+  overflow: hidden;
+  padding-bottom: 56.25%; /* 56.25% = 16x9 */
```

Comment:
You don't need to do this trick anymore, we can use the native `aspect-ratio` (https://caniuse.com/?search=aspect).

So this would be:
```
.video__wrap {
  position: relative;
  aspect-ratio: 16 / 9;
}
```
and it should work ©️.